src/app/components/login/login.component.ts

```diff
@@ -26,6 +28,34 @@ export class LoginComponent {
 
   matcher = new ErrorMatcher();
 
+  constructor(private loginService: LoginService, private userService: UserService) {}
+
+  sendDecodeFormData() {
+    let uname: string = "";
+    let pass: string = "";
+    if (this.form.controls['user'].value !== null) {
+       uname = this.form.controls['user'].value;
+    }
+    if (this.form.controls['password'].value !== null) {
+      pass = this.form.controls['password'].value;
+    }
+    console.log(this.form.controls['user']);
+    this.loginService
+      .sendFormInfo(
+        this.encodeDataBase64(uname),
+        this.encodeDataBase64(pass)
+      )
+      .subscribe({
+        next: (response: any) => {
```

Comment:
```suggestion
        next: (response: ResponseInterface) => {
```

Declare an interface to describe que object recieved